libsolidity/lsp/LanguageServer.cpp

```diff
@@ -210,6 +211,39 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		int typeFailureCount = 0;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (jsonPath.isObject())
+				{
+					remappings.emplace_back(ImportRemapper::Remapping{
+						jsonPath.get("context", "").asString(),
+						jsonPath.get("prefix", "").asString(),
+						jsonPath.get("target", "").asString(),
+					});
+				}
+				else
+				{
+					typeFailureCount++;
+				}
+			}
+			m_fileRepository.setRemappings(move(remappings));
+		}
+		else
+			++typeFailureCount;
```

Comment:
This number is not used to `break` the loop at any point and only used as a `bool` later, so I think it should be a bool instead.